src/training_hub/algorithms/lora_grpo_verl.py

```
@@ -0,0 +1,1077 @@
+            "data.filter_overlong_prompts=True",
+            "data.truncation=error",
+            # Model + LoRA
+            f"actor_rollout_ref.model.path={model_path}",
+            f"actor_rollout_ref.model.lora_rank={lora_r}",
+            f"actor_rollout_ref.model.lora_alpha={lora_alpha}",
+            "actor_rollout_ref.model.enable_gradient_checkpointing=True",
+            "actor_rollout_ref.model.use_remove_padding=True",
+            # Actor (training)
+            f"actor_rollout_ref.actor.optim.lr={learning_rate}",
+            f"actor_rollout_ref.actor.ppo_mini_batch_size={min(train_batch_size, 256)}",
+            f"actor_rollout_ref.actor.ppo_micro_batch_size_per_gpu=1",
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)